packages/react-components/react-components/src/Concepts/Icons/IconTable.stories.mdx

```diff
@@ -0,0 +1,10 @@
+import { Meta } from '@storybook/addon-docs';
+import { Spinner } from '@fluentui/react-spinner';
+
+<Meta title="Concepts/Developer/Icons/IconTable" />
+
+### Available Icons
+
+<React.Suspense fallback={<Spinner label="Loading..." />}>
+  {React.createElement(React.lazy(() => import('./ReactIconGrid')))}
+</React.Suspense>
```

Comment:
It's not related to this PR, but I noticed few things:
- We use unsized icons internally, but I don't see a radio item to select them. Is it expected?
- Search is not a great experience as we don't display icon names: 
![image](https://user-images.githubusercontent.com/14183168/183606826-d7893efc-3b5b-4a64-8e67-dfa78f2386ce.png)

Reply:
For a reference, you can also check https://fluentsite.z22.web.core.windows.net/0.63.0/icon-viewer.

Reply:
![image](https://user-images.githubusercontent.com/66456876/186523406-ae2cee70-0fee-4b2b-87f2-097032d55c42.png)